crates/payout_curve/examples/payout_curve_csv.rs

```diff
@@ -27,7 +27,9 @@ fn main() -> Result<()> {
     let quantity = 30_000.0;
     let leverage_short = 3.0;
     let leverage_long = 2.0;
-    let fee = 0;
+    // fee is 0.3% * quantity/initial_price = 0.003 BTC
+    let fee = 0.003 * initial_price / quantity;
+    dbg!(fee);
```

Comment:
🔧 i guess you did not intent to commit that.

Reply:
nope 🙈